src/main/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/DataflowReadsPipeline.java

```diff
@@ -103,26 +105,29 @@ final protected void setupPipeline(Pipeline pipeline) {
     }
 
     @VisibleForTesting
-    protected PCollection<?> applyTransformsToPipeline(String headerString, PCollection<Read> preads) {
-        for (ReadFilter filter : getReadFilters()) {
+    protected PCollection<?> applyTransformsToPipeline(final String headerString, final PCollection<Read> preadsArgs) {
```

Comment:
I don't understand this rename.  This thing coming in is actually the PCollection of Reads, not arguments.